scripts/linters/test_files/invalid_import_order.py

```diff
@@ -26,9 +26,9 @@
 import fnmatch  # pylint: disable=unused-import
 import multiprocessing  # pylint: disable=unused-import
 import os  # pylint: disable=unused-import
+import threading  # pylint: disable=unused-import
```

Comment:
Why did this change?

Reply:
It is needed in order to it to be invalid, (it was also invalid because `python_utils` were in wrong group, but I fixed that now).